src/server/editionsPage.tsx

```diff
@@ -89,9 +92,16 @@ const buildHtml = (head: string, body: string): string => `
 function render(imageSalt: string, request: RenderingRequest): Page {
 	const item = fromCapi({ docParser, salt: imageSalt })(request);
 	const body = renderBody(item);
+	const head = renderHead(
+		request,
+		getThirdPartyEmbeds(request.content),
+		body.css,
+		body.ids,
+		requiresInlineStyles(request.content),
```

Comment:
Oh! I didn't realise this had been activated... That's not good, it [applies `unsafe-inline`](https://github.com/guardian/apps-rendering/blob/bbf196d8665902d760f7b3f405f2455203276817/src/server/csp.ts#L84) to our `style-src`.

Please don't rely on `unsafe-inline` to be available, I think I'm going to turn it off again for now.

Reply:
#1025